packages/fxa-payments-server/src/lib/stripe.ts

```diff
@@ -109,6 +113,7 @@ export async function handleSubscriptionPayment({
         priceId: selectedPlan.plan_id,
         productId: selectedPlan.product_id,
         idempotencyKey,
+        promotionCode: promotionCode || undefined,
```

Comment:
nit: ` || undefined` is unnecessary since `promotionCode` has the type of  `string | undefined`.

Reply:
removed